### DISCUSSION TITLE
Having a Future Steps Section

### DISCUSSION THREAD

**Discussion by behradkhadem** [maxspahn/gym_envs_urdf#229] [Ideas]
This package is really state of the art and universal at the same time and I haven't found any better alternative for this package, really! (Special thanks to its contributors and developers 💖) But one area that this project lacks is not having a specific roadmap, so that others could contribute. A static page or a section inside the `readme.md` file would suffice in my opinion. 
These are some important future steps IMHO:

- **DOCS!** Having a well-written and comprehensive documentation is a MUST for every package, specially this one. I understand that this is has to be done after the initial stable release (`v 1.0.0`) but I don't think that basics change until then. (I prefer [docsify](https://[docsify](https://docsify.js.org/#/).js.org/#/))
- **Having a catchier name.** This package really lacks in branding! A catchier more concise name can increase the userbase and possible candidates for contributing to the project.  
- **Extracting the physics simulator and supporting other engines as well**. I know that this is a huge deal and will take a lot of time but it's possible and necessary in my opinion. Imagine that we could give user the option to choose its simulator (use MuJoCo or UE5 instead of pyBullet)!

**Comment by behradkhadem** [comment]
@maxspahn 😅

**Reply by maxspahn** [reply]
I agree to all, but I don't have the time to do it at the moment. Also the making urdfenvs big is not my main goal here
